client/src/components/Header/Header.style.js

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+export const H1 = styled.h1`
+  color: #fff;
+  font-size: 28px;
+  padding: 0;
+  margin: 0;
+  height: 36px;
+  img {
+    height: 56px;
+  }
+`;
+
+export const H2 = styled.h2`
+  color: #000;
+`;
+
+export const User = styled.img`
+  height: 20px;
+  padding: 18px 0 0 20px;
+`;
+
+export const ShoppingBasket = styled.img`
+  height: 20px;
+  padding: 18px 20px 0 0px;
+  text-align: right;
+`;
+
+export const StyledHeader = styled.header`
+  background: #15b7d3;
```

Comment:
A thought. I think it would make sense to have a colour scheme file where we store all commonly used colours. We could import them when we need them. That way we know that we are staying consistent, also in the future if we want to change things up we can do it easily.

Reply:
The same with fonts? We save 1 for headers and 1 for plain text. We can change font appropriately in seconds that way

Reply:
I've not seen a way to do this with Styled Components that scales well - can you send me an example?

Reply:
Here is a guide on how to do it in normal css: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables

Some information on how to use it with component style.

I think the way to do it would be setting the variables in the App level, then they should automatically be inherited to the lower levels. Worth checking with Jem if this is the best way to do it/ if we will run in to any issues with browsers like IE.